static/js/libs/fileValidation.test.js

```diff
@@ -379,7 +379,105 @@ describe("validateRestrictions", () => {
         });
 
         expect(validation.errors).toEqual([
-          "has a width (10 pixels) that is 0.33x its height (30 pixels). Its width needs to be between 0.5x and 0.33x the height."
+          "(10 x 30 pixels) does not have the correct aspect ratio: it needs to be between 1:2 and 1:3"
```

Comment:
10x30 seems like it is 1:3 (so fits in range)... why do we expect it to error?

Reply:
Yeah, I've made the logic better for this. Rather than assuming `min` and `max` aspect ratios (in the restrictions config) are always the correct way round it now checks that the ratio is between or equal to the two ratios.